setup/destroy.py

```diff
@@ -0,0 +1,22 @@
+from setup import *
+import dotenv
+import sys
+
+
+def destroy_database():
```

Comment:
Is this code/file required for Mongo to operate? I can't image we'll ever be calling it from production.

Reply:
Yeah won't ever be called in production, was just for resetting the dev environment.